plugins/services/diff/local.go

```diff
@@ -41,7 +41,8 @@ type config struct {
 	// respected for which is chosen. Each differ should return the same
 	// correct output, allowing any ordering to be used to prefer
 	// more optimimal implementations.
-	Order []string `toml:"default"`
+	Order  []string `toml:"default"`
+	SyncFs bool     `toml:"sync_fs"`
```

Comment:
This could use a doc string but overall I think this makes sense.

Reply:
done